app/javascript/components/mentoring/discussion/AddDiscussionPost.tsx

```diff
@@ -11,22 +12,29 @@ export const AddDiscussionPost = ({
   onSuccess?: () => void
 }): JSX.Element => {
   const [open, setOpen] = useState(false)
+  const [value, setValue] = useState('')
 
   const handleSuccess = useCallback(() => {
-    onSuccess()
-
     setOpen(false)
+    setValue('')
+    onSuccess()
   }, [onSuccess])
 
   return (
     <section className="comment-section">
+      <button className="new-messages-button" type="button">
+        <GraphicalIcon icon="comment" />
+        <span>1 New Message</span>
+      </button>
```

Comment:
@kntsoriano This should go here. I've put the right HTML here. The existing code with the hasNewMessages and stuff will need to be refactored to work here. You could also move the "comment-section" div into the parent class (Discussion.tsx), and move this in there. 

The key is that `.new-messages-button` needs to live within `.comment-section`